code/modules/plumbing/plumbers/fermenter.dm

```diff
@@ -0,0 +1,53 @@
+/obj/machinery/plumbing/fermenter
+	name = "chemical fermenter"
+	desc = "chemical fermetner."
```

Comment:
ok

Reply:
Just a chemical fermetner. Nothing to see here.
```suggestion
	desc = "Turns plants into various types of booze."
```